### ISSUE TITLE
make notebook (colab) examples work on TPUs

### ISSUE THREAD

**Issue by fabianp** [google/jaxopt#202]
these are the examples on docs/notebooks/

Probably it's necessary to add this line when running on TPUs https://github.com/google/jax#pip-installation-colab-tpu

I think this would be a good first issue

**Comment by Samad999777** [comment]
Hey @fabianp is this still open for contribution ?

**Comment by fabianp** [comment]
yes!

**Comment by Samad999777** [comment]
Great, I would like to work on it

**Comment by fabianp** [comment]
all yours :-)

**Comment by Samad999777** [comment]
I actually have to reproduce the already given examples in colab notebooks using TPU?

**Comment by fabianp** [comment]
yes, the goal is that the examples run and produce similar results using TPU instead of GPU. Some might work out of the box, some might fail (nobody knows because nobody has tried yet)

**Comment by Samad999777** [comment]
Alright, I will keep you posted about it. You can assign this issue to me.

**Comment by Samad999777** [comment]
Hey @fabianp I have a colab notebook working for one example. Should i create a pull request so you can check it out?

**Comment by fabianp** [comment]
Yes please :-)

**Comment by fabianp** [comment]
@Samad999777 : any updates on this :-) ?

**Comment by fabianp** [comment]
There's no rush, I was reaching out mostly in case you needed some help.
Again, there's no urgency in this

**Comment by Samad999777** [comment]
@fabianp thanks alot, hopefully we will wrap it up this week.

**Comment by rosspeili** [comment]
Anyone solved this?
